packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio/gux-form-field-radio.tsx

```diff
@@ -36,10 +36,16 @@ export class GuxFormFieldRadio {
   private disabled: boolean;
 
   @State()
-  private hasError: boolean = false;
+  private hasHelp: boolean = false;
 
   @State()
-  private hasHelp: boolean = false;
+  private hasError: boolean = false;
+
+  @Prop()
+  hasGroupError: boolean = false;
+
+  @Prop()
+  hasGroupDisabled: boolean = false;
```

Comment:
Thoughts on naming this `isGroupDisabled` instead?

Reply:
I think that keeping the names to `hasGroupError` and `hasGroupDisabled` helps keep it clear that these are set by the parent `group` component and has values that are separate from the radio button's own error and disabled state.